test/ui/ViewJobRolesTests.ts

```diff
@@ -0,0 +1,62 @@
+import { ViewJobRolesTestsPage } from '../ui/ViewJobRolesTestsPage';
+import { HeaderTestPage } from '../ui/HeaderTestPage';
+import { FooterTestPage } from '../ui/FooterTestPage';
+import { basepage } from './basepage';
+
+describe('View Job Roles Tests', function () {
+    this.timeout(70000);
```

Comment:
is this needed?

Reply:
It doesn't work without this as the tests timeout too quickly